### PR TITLE
Move TOC meetings to wednesdays

### PR DESCRIPTION
**Pull request by psschwei**
# Changes

Moving TOC meetings from Thursdays to Wednesdays. Discussed and agreed to by TOC in Slack on January 9.

cc @knative/technical-oversight-committee

### CODE REVIEW

**Comment by psschwei** [comment]
We need @knative/steering-committee to approve the change in the top-level TOC doc

File: TECH-OVERSIGHT-COMMITTEE.md [hunk 1]
```diff
@@ -84,7 +84,7 @@ during the community feedback period at the end of each meeting.
 | -------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | Google Group               | [knative-tech-oversight@googlegroups.com](https://groups.google.com/forum/#!forum/knative-tech-oversight)                                                   |
 | Community Meeting VC       | See the top of the [Meeting notes](https://docs.google.com/document/d/1LzOUbTMkMEsCRfwjYm5TKZUWfyXpO589-r9K2rXlHfk/edit)                                    |
-| Community Meeting Calendar | Thursdays at 8:30a-9:15a <br>[Calendar](https://calendar.google.com/calendar/embed?src=knative.team_9q83bg07qs5b9rrslp5jor4l6s%40group.calendar.google.com) |
```
Comment (dprotaso) [inline TECH-OVERSIGHT-COMMITTEE.md:87]: do we want to change the time to 10 EST?

Comment (psschwei) [reply TECH-OVERSIGHT-COMMITTEE.md:87]: good catch

**Comment by dprotaso** [comment]
LGTM

**Pull request merged**